spie-10707-10.tex

```diff
@@ -28,6 +29,8 @@
 \affil[c]{Centre de Calcul de l'IN2P3, USR 6402 du CNRS/IN2P3, 43 Bd. du 11 Novembre 1918, 69622 Villeurbanne Cedex, France}
 \affil[d]{SLAC National Accelerator Laboratory, 2575 Sand Hill Rd, Menlo Park, CA 94025, USA}
 \affil[e]{Department of Astrophysical Sciences, Princeton University, Princeton, NJ 08544}
+\affil[f]{NCSA, University of Illinois at Urbana-Champaign, 1205 W. Clark St. Urbana, IL 61801}
```

Comment:
I think you are technically "e" affiliation because you come before Pim.

Reply:
fixed